tgui/packages/tgui/interfaces/NtosCyborgRemoteMonitor.jsx

```diff
@@ -58,23 +58,25 @@ export const NtosCyborgRemoteMonitorContent = (props) => {
             selected={tab_main === 1}
             onClick={() => setTab_main(1)}
           >
-            Cyborgs
+            Борги
           </Tabs.Tab>
           <Tabs.Tab
             icon="clipboard"
             lineHeight="23px"
             selected={tab_main === 2}
             onClick={() => setTab_main(2)}
           >
-            Stored Log File
+            Сохраненные логи борга
           </Tabs.Tab>
         </Tabs>
       </Stack.Item>
       {tab_main === 1 && (
         <>
           {!card && (
             <Stack.Item>
-              <NoticeBox>Certain features require an ID card login.</NoticeBox>
+              <NoticeBox>
+                Некоторые функции требуют входа в систему с помощью ID-карты.
+              </NoticeBox>
```

Comment:
```suggestion
              <NoticeBox>Некоторые функции требуют ввод ID-карты.</NoticeBox>
```
Пытаемся не злить Prettier

Reply:
тут вероятно оно само так получилось. Текст длинный мб